### ISSUE TITLE
Add "base class" functionality.

### ISSUE THREAD

**Issue by alwaysblank** [alwaysblank/climber#6]
Currently the user can set classes relatively easily, but if the user wants to change several classes in the same way (i.e., adjust the "root" part of the default classes—`simpleMenu`) then they have to specify a new class for each class property. This is unwieldy, and will only be worse if we end up adding more class properties. There should be a way to have these classes processed at runtime, which would allow the user to specify a new 'base' class.

An example of this might look like:

```php
$Climber = new Climber( // imagine a correct instantiation here );
$Climber->cssBase = 'complexMenu';
echo $Climber->topClass;
echo $Climber->menuClass;
// 'complexMenu'
// 'complexMenu__menu'
// etc...
```

**Comment by alwaysblank** [comment]
The obvious solution seems like it would involve two things:

- A new `baseClass` property.
- (s)printf-formatted strings for default classes, i.e. `%s__menu`, `%s__item`, etc.

During the final step, as Climber generates the HTML, these strings would be run through (s)printf, evaluating them against the `baseClass` property.

This way, users would specify only a new base class, if that's all that's required, but could also directly over-write classes if necessary.

**Comment by alwaysblank** [comment]
Resolved with commit da04b6930fd132fe5f22902ecf010554049aca97. Users can change the base class with the syntax described above, but the property to modify is called `baseClass` (not `cssBase`).
